beacon-chain/rpc/eth/validator/handlers.go

```diff
@@ -518,11 +519,11 @@ func (s *Server) produceSyncCommitteeContribution(
 		http2.HandleError(w, "Could not get contribution data: "+err.Error(), http.StatusInternalServerError)
 		return nil, false
 	}
-	return &ethpbv2.SyncCommitteeContribution{
```

Comment:
Since this whole code is in the handler, we should return `shared.SyncCommitteeContribution`. No need to use v1alpha1 types in the handler unless we use core functions that take in or return them.